src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseController.cs

```diff
@@ -229,6 +254,26 @@ private async Task<ActionResult> CheckReleaseExistsAsync(ReleaseId releaseId, Fu
 
             return Ok(result.Right);
         }
+        
+        private async Task<ActionResult> CheckReleaseExistsStreamAsync(ReleaseId releaseId,
+            Func<Task<Either<ValidationResult, FileStreamResult>>> andThen)
+        {
+            var release = await _releaseService.GetAsync(releaseId);
+            if (release == null)
+            {
+                return NotFound();
```

Comment:
I think this might want to be bad request or no content

Reply:
Here it is not that the file cannot be found its that the release cannot be found (though I guess as a result of that the file can't be found). Do you still want it changing?